src/content/protocols/ethena.md

```diff
@@ -0,0 +1,158 @@
+
+> Chain score: L
+
+## Upgradeability
+
+Ethena's smart contracts are not upgradable but the ability to set and update roles introduces a level of flexibility and risk. 
+
+> Upgradeability: L
+
+## Autonomy
+
+Ethena's contracts have dependencies on external roles and addresses, such as the minter, redeemer, and custodian addresses. 
+
+> Autonomy score: H
+
+## Exit Window
+
+Ethena's contracts have processes for transferring admin rights and setting role without any exit window. 
+
+> Exit Window score: H
+
+## Accessibility
+
+Only a single user interface exists.
+
+> Accessibility score: M
+
+# Technical Analysis
+
+## Contracts
+
+| Contract Name | Address |
+| ------------- | ------- |
+| USDe    | [0x4c9EDD5852cd905f086C759E8383e09bff1E68B3](https://etherscan.io/address/0x4c9EDD5852cd905f086C759E8383e09bff1E68B3)   |
+| EthenaMinting    | [0xe3490297a08d6fC8Da46Edb7B6142E4F461b62D3](https://etherscan.io/address/0xe3490297a08d6fC8Da46Edb7B6142E4F461b62D3)   |
+| StakedUSDeV2  | [0x9D39A5DE30e57443BfF2A8307A4256c8797A3497](https://etherscan.io/address/0x9D39A5DE30e57443BfF2A8307A4256c8797A3497)    |
+| StackingRewardDistributor | [0xf2fa332bD83149c66b09B45670bCe64746C6b439](https://etherscan.io/address/0xf2fa332bd83149c66b09b45670bce64746c6b439) |
+| ENA | [0x57e114B691Db790C35207b2e685D4A43181e6061](https://etherscan.io/address/0x57e114B691Db790C35207b2e685D4A43181e6061) |
+| sENA | [0x8bE3460A480c80728a8C4D7a5D5303c85ba7B3b9](https://etherscan.io/token/0x8bE3460A480c80728a8C4D7a5D5303c85ba7B3b9) |
+
+## Permission owners
+
+Ethena utilises a gnosis safe multisig to hold ownership of its smart contracts. All multisig keys are cold wallets and each multisig requires 7/10 or more confirmations before transactions are approved. This multisig is purely for the purpose of owning the smart contracts, and will not hold funds or do other on chain actions.
+
+
+| Name | Account                                       | Type         |
+| ---- | --------------------------------------------- | ------------ |
+| Dev   | [0x3B0AAf6e6fCd4a7cEEf8c92C32DFeA9E64dC1862](https://etherscan.io/address/0x3B0AAf6e6fCd4a7cEEf8c92C32DFeA9E64dC1862) | Multisig 4/8|
+| Hot Swap | [0x4423198f26764a8ce9ac8f1683c476854c885d9d](https://etherscan.io/address/0x4423198f26764a8ce9ac8f1683c476854c885d9d) | Multisig 4/8 |
+| sUSDe Payout Fund | [0x71e4f98e8f20c88112489de3dded4489802a3a87](https://etherscan.io/address/0x71e4f98e8f20c88112489de3dded4489802a3a87) | Multisig 4/8 |
+| Protocol Treasury | [0xa2af0b03aaf167cfc9624c6ef587581b6fcced92](https://etherscan.io/address/0xa2af0b03aaf167cfc9624c6ef587581b6fcced92) | Multisig 5/9 |
+| Trading Operations | [0x0a0b96A730ED5CDa84bcB63c1Ee2edCb6B7764d6](https://etherscan.io/address/0x0a0b96A730ED5CDa84bcB63c1Ee2edCb6B7764d6) | Multisig 4/8 |
+| Reserve Fund | [0x2b5ab59163a6e93b4486f6055d33ca4a115dd4d5](https://etherscan.io/address/0x2b5ab59163a6e93b4486f6055d33ca4a115dd4d5) | Multisig 4/8 |
```

Comment:
can you please assign them to the functions where they own permissions

Reply:
Solved